packages/common/src/types/errors.ts

```diff
@@ -0,0 +1,7 @@
+import { RpcError } from '@protobuf-ts/runtime-rpc';
+
+export { RpcError };
+
+export function isRpcError(error: unknown): error is RpcError {
+    return (error as RpcError).code !== undefined;
+}
```

Comment:
I think we should check it a little more thorough here :thinking: 
Looking at the type is also always has a meta field and a name field (which seems to always be "RpcError")

Reply:
I agree, I'll check all other required fields as well

Reply:
And add it to the documentation